net/MapViewController.m

```diff
@@ -478,13 +433,6 @@ -(void)animateTabsSwitching:(UIViewController *)viewController
     
     UIView *fromView = self.tabBarController.selectedViewController.view;
     UIView *toView = [viewController view];
-    //UIView *toView = [self.tabBarController.viewControllers[controllerIndex] view];
```

Comment:
Unfortunatelly, for now, we don't have generall opinion about using (or not) animation with changing viewcontrollers with tabbar.